Gruntfile.js

```diff
@@ -15,23 +15,49 @@ module.exports = function(grunt) {
         serverToken: '<%= secrets.serverToken %>'
       },
       email: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
+        from: '<%= secrets.testSender %>',
+        to: '<%= secrets.testRecipient %>',
         subject: 'Yo',
         src: ['test/email.html']
       },
       bulk: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
+        from: '<%= secrets.testSender %>',
+        to: '<%= secrets.testRecipient %>',
         subject: 'Hey',
         src: ['test/*.html']
       }
-    }
+    },
+
+    "postmark-templates": {
```

Comment:
Having separate tasks for pushing new templates and editing existing templates might be hard to manage for users. I imagine this would work as a single task(`grunt sync`), where it creates a new template if the referenced template isn't found on the server, otherwise it updates the existing template. That way users don't have to worry about what lives on the server and what doesn't.